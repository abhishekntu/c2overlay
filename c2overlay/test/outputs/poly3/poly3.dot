digraph graphname {
N12 [color="black", label="N8_mul_Imm_80_N8_N12"];
N13 [color="black", label="N9N7_mul_Imm_2_N9_add_N7_N13"];
N10 [color="black", label="N16_sqr_N16_N10"];
N11 [color="black", label="N15N13N14_sub_N15_mul_Imm_15_N13_add_N14_N11"];
N14 [color="black", label="O0_N18_N14"];
N8 [color="black", label="N10_mul_Imm_11_N10_N8"];
N9 [color="black", label="N11N17_mul_Imm_25_N11_sub_N17_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="I3_N4_N4"];
N5 [color="black", label="I4_N5_N5"];
N6 [color="black", label="I5_N6_N6"];
N7 [color="black", label="N12_mul_N12_N7"];
N12 -> N9;
N13 -> N11;
N10 -> N8;
N11 -> N14;
N8 -> N11;
N9 -> N11;
N1 -> N13;
N2 -> N7;
N3 -> N10;
N4 -> N7;
N5 -> N13;
N6 -> N12;
N7 -> N9;
}
