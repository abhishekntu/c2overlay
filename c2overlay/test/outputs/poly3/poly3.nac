procedure foo(in s32 t,in s32 u,in s32 v,in s32 x,in s32 y,in s32 z,out s32 D_2451)
{
  localvar s32 D_2450;
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= sqr v;
  t1_2 <= mul y,u;
  t2_1 <= mul t1_1,11;
  t2_2 <= mul t1_2,25;
  D_2450 <= mul t,2;
  t3_1 <= add D_2450,x;
  t3_2 <= mul z,80;
  t4_1 <= sub t3_1,t2_1;
  t4_2 <= sub t2_2,t3_2;
  t5_1 <= mul t4_1,15;
  D_2451 <= add t5_1,t4_2;
}
