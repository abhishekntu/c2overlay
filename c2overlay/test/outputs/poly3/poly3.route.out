Array size: 5 x 5 logic blocks.

Routing:

Net 0 (N1)

SOURCE (0,1)  Pad: 1  
  OPIN (0,1)  Pad: 1  
 CHANY (0,1)  Track: 1  
  IPIN (1,1)  Pin: 3  
  SINK (1,1)  Class: 0  


Net 1 (N2)

SOURCE (6,1)  Pad: 1  
  OPIN (6,1)  Pad: 1  
 CHANY (5,1)  Track: 1  
  IPIN (5,1)  Pin: 1  
  SINK (5,1)  Class: 0  


Net 2 (N3)

SOURCE (3,0)  Pad: 1  
  OPIN (3,0)  Pad: 1  
 CHANX (3,0)  Track: 1  
 CHANY (2,1)  Track: 0  
 CHANY (2,2)  Track: 0  
  IPIN (3,2)  Pin: 3  
  SINK (3,2)  Class: 0  


Net 3 (N4)

SOURCE (5,0)  Pad: 1  
  OPIN (5,0)  Pad: 1  
 CHANX (5,0)  Track: 1  
  IPIN (5,1)  Pin: 2  
  SINK (5,1)  Class: 0  


Net 4 (N5)

SOURCE (1,0)  Pad: 1  
  OPIN (1,0)  Pad: 1  
 CHANX (1,0)  Track: 1  
  IPIN (1,1)  Pin: 2  
  SINK (1,1)  Class: 0  


Net 5 (N6)

SOURCE (4,0)  Pad: 1  
  OPIN (4,0)  Pad: 1  
 CHANX (4,0)  Track: 1  
  IPIN (4,1)  Pin: 2  
  SINK (4,1)  Class: 0  


Net 6 (N12)

SOURCE (4,1)  Class: 1  
  OPIN (4,1)  Pin: 7  
 CHANY (3,1)  Track: 1  
  IPIN (3,1)  Pin: 1  
  SINK (3,1)  Class: 0  


Net 7 (N13)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 5  
 CHANY (1,1)  Track: 1  
  IPIN (2,1)  Pin: 3  
  SINK (2,1)  Class: 0  


Net 8 (N10)

SOURCE (3,2)  Class: 1  
  OPIN (3,2)  Pin: 7  
 CHANY (2,2)  Track: 1  
  IPIN (2,2)  Pin: 1  
  SINK (2,2)  Class: 0  


Net 9 (N14)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 6  
 CHANX (2,0)  Track: 1  
  IPIN (2,0)  Pad: 0  
  SINK (2,0)  Pad: 0  


Net 10 (N8)

SOURCE (2,2)  Class: 1  
  OPIN (2,2)  Pin: 6  
 CHANX (2,1)  Track: 1  
  IPIN (2,1)  Pin: 0  
  SINK (2,1)  Class: 0  


Net 11 (N9)

SOURCE (3,1)  Class: 1  
  OPIN (3,1)  Pin: 7  
 CHANY (2,1)  Track: 1  
  IPIN (2,1)  Pin: 1  
  SINK (2,1)  Class: 0  


Net 12 (N7)

SOURCE (5,1)  Class: 1  
  OPIN (5,1)  Pin: 4  
 CHANX (5,1)  Track: 1  
 CHANX (4,1)  Track: 1  
 CHANX (3,1)  Track: 1  
  IPIN (3,1)  Pin: 0  
  SINK (3,1)  Class: 0  
